packages/explorer/src/utils.ts

```diff
@@ -2,6 +2,12 @@ import { formatEther } from "viem";
 import { type ClassValue, clsx } from "clsx";
 import { twMerge } from "tailwind-merge";
 
+Object.defineProperty(BigInt.prototype, "toJSON", {
+  get() {
+    return () => String(this);
+  },
+});
```

Comment:
no no no no no no, please don't do this, the web has a long long history of learning this lesson over and over and this is always a bad idea

viem has a `stringify` util to stringify JSON with bigints if you wanna use that instead: https://github.com/wevm/viem/blob/main/src/utils/stringify.ts

Reply:
yeah my bad, I was taking the approach here of "master the rules before breaking them" but probably it's like with "don't roll your own crypto", just don't do it. `stringify` looks perfect, thanks, updating.

Reply:
specifically extending native prototypes that is the bad idea!

Reply:
yeah actually looking back it was a rather terrible idea, as you said should just never do it. Wish I knew about `stringify` before, very useful! Added it.